ci/docs/build.sh

```diff
@@ -18,27 +18,27 @@ export LIBCUDF_KERNEL_CACHE_PATH="$HOME/.jitify-cache"
 export NIGHTLY_VERSION=$(echo $BRANCH_VERSION | awk -F. '{print $2}')
 export PROJECTS=(cusignal)
 
-logger "Check environment..."
+gpuci_logger "Check environment..."
 env
 
-logger "Check GPU usage..."
+gpuci_logger "Check GPU usage..."
 nvidia-smi
 
-logger "Activate conda env..."
+gpuci_logger "Activate conda env..."
 source activate rapids
 # TODO: Move installs to docs-build-env meta package
 conda install -c anaconda markdown beautifulsoup4 jq
 pip install sphinx-markdown-tables
```

Comment:
```suggestion
# TODO: Move installs to docs-build-env meta package
conda install -c anaconda markdown beautifulsoup4 jq
pip install sphinx-markdown-tables
```

similarly to the comment on Mike's PR below, this whole `TODO` block can be removed. I see `sphinx-markdown-tables`, `markdown` and `beautifulsoup4` in the [rapids-doc-env](https://github.com/rapidsai/integration/blob/branch-0.16/conda/recipes/rapids-doc-env/meta.yaml#L37-L49) and `jq` in our [gpuci devel images](https://github.com/rapidsai/gpuci-build-environment/blob/branch-0.16/rapidsai/devel.Dockerfile#L80-L83).

https://github.com/rapidsai/cudf/pull/6329#discussion_r506506128

Reply:
Done!